core/modules/css/CssAccess.cc

```diff
@@ -467,74 +467,16 @@ MatchTableParams
 CssAccess::getMatchTableParams(std::string const& dbName,
                                std::string const& tableName) const {
     LOGS(_log, LOG_LVL_DEBUG, "getMatchTableParams(" << dbName << ", " << tableName << ")");
-    _checkVersion();
-
-    std::string const tableKey = _prefix + "/DBS/" + dbName + "/TABLES/" + tableName;
-
-    MatchTableParams params;
-
-    std::vector<std::string> subKeys{"dirTable1", "dirColName1", "dirTable2", "dirColName2", "flagColName"};
-    auto paramMap = _getSubkeys(tableKey + "/match", subKeys);
-    if (paramMap.empty()) {
-        // check table key
-        if (not _kvI->exists(tableKey)) throw NoSuchTable(dbName, tableName);
-        return params;
-    }
-
-    // fill the structure
-    params.dirTable1 = paramMap["dirTable1"];
-    params.dirColName1 = paramMap["dirColName1"];
-    params.dirTable2 = paramMap["dirTable2"];
-    params.dirColName2 = paramMap["dirColName2"];
-    params.flagColName = paramMap["flagColName"];
-
-    return params;
+    TableParams params = getTableParams(dbName, tableName);
+    return params.match;
```

Comment:
No, please don't do this. Potentially you are reading (or trying to read) much more than you need here but then throw away big fraction of it. This code is supposed to be very efficient (it's executed by czar for every table in a query), we need to avoid possible overheads. The only reason that this separate method exist is to make its implementation more efficient, otherwise everyone could just use `getTableParams()`.